vendor/calcite-1_19_0/build.gradle

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins { id 'org.apache.beam.vendor-java' }
+
+description = "Apache Beam :: Vendored Dependencies :: Calcite 1.19.0"
+
+group = "org.apache.beam"
+version = "0.1"
+
+def calcite_version = "1.19.0"
+def avatica_version = "1.15.0"
+def prefix = "org.apache.beam.vendor.calcite.v1_19_0"
+
+vendorJava(
+  dependencies: [
+    "org.apache.calcite:calcite-core:$calcite_version",
+    "org.apache.calcite:calcite-linq4j:$calcite_version",
+    "org.apache.calcite.avatica:avatica-core:$avatica_version",
+  ],
+  relocations: [
+    "org.apache.calcite": "${prefix}.org.apache.calcite",
+
+    // Calcite has Guava on its API surface
+    "com.google.common": "org.apache.beam.vendor.guava.v26_0_jre.com.google.thirdparty",
```

Comment:
I don't understand this. Why are we moving "com.google.common" to "com.google.thirdparty"? This does not match guava: https://github.com/apache/beam/blob/f7cbf88f550c8918b99a13af4182d6efa07cd2b5/vendor/guava-26_0-jre/build.gradle#L29

Reply:
address this.